FIX

https://github.com/Jhon-Crow/real-time-voice-replacement/actions/runs/20558545931/job/59046049463

Error on line 79 in D:\a\real-time-voice-replacement\real-time-voice-replacement\installer.iss: Column 83:
Unknown identifier 'ResultCode'
Compile aborted.
Error: Process completed with exit code 2.

--- a/CLAUDE.md
+++ b/CLAUDE.md
@@ -1,7 +1,0 @@
-Issue to solve: https://github.com/Jhon-Crow/real-time-voice-replacement/issues/6
-Your prepared branch: issue-6-f3fe2cb96371
-Your prepared working directory: /tmp/gh-issue-solver-1766950791202
-Your forked repository: konard/Jhon-Crow-real-time-voice-replacement
-Original repository (upstream): Jhon-Crow/real-time-voice-replacement
-
-Proceed.

